examples/docker-compose.yml

```
@@ -2,13 +2,20 @@ version: '3.4'
 
 services:
   bds:
-    image: itzg/minecraft-bedrock-server
+    image: foleymic/minecraft-bedrock-server
     environment:
       EULA: "TRUE"
       GAMEMODE: survival
       DIFFICULTY: normal
+      SERVER_PORT: 19140
+    healthcheck:
+      test: CMD /mc-monitor status-bedrock --host 127.0.0.1 --port 19132 | grep online
```

No need to grep the output since exit code already indicates healthiness
```suggestion
      test: CMD /mc-monitor status-bedrock --host 127.0.0.1 --port 19132
```